.github/workflows/CI.yml

```diff
@@ -0,0 +1,27 @@
+name: Run Tests 
+
+on: 
+  push:
+    branches:
```

Comment:
Should we add `main` branch here as well?

Reply:
We can but this repo main branch is `master` so there is not impact.

Reply:
Done anyway 😄